### PR TITLE
Flag local delivery

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

**Comment by paulgrever** [comment]
@spencern  another one!

File: client/templates/fulfillmentOrders/fulfillmentOrders.js [hunk 1]
```diff
@@ -38,9 +44,10 @@ Template.fulfillmentOrder.helpers({
     return moment(longDate).format('MMMM Do, YYYY');
   },
   toBeShipped: function () {
-    let fullRoute = Iron.Location.get().path;
-    let thisRoute = fullRoute.substr(32, 8);
-    if (thisRoute === 'shipping') {
+    let fullRoute = Router.current().url;
```
Comment (spencern) [inline client/templates/fulfillmentOrders/fulfillmentOrders.js:47]: potentially easier would be using `Router.current().route.getName()`

File: server/publications/afOrders.js [hunk 2]
```diff
@@ -14,7 +14,10 @@ Meteor.publish('afOrders', function () {
         'email': 1,
         'shopifyOrderNumber': 1,
         'shipping.address.phone': 1,
-        'history': 1
+        'history': 1,
```
Comment (spencern) [inline server/publications/afOrders.js:17]: Going to add a sev3 card to review our publications and subscriptions at some point. We may do better with more varied publications in the future.

**Comment by spencern** [comment]
A few notes, but looks good. Nice work.

**Pull request merged**